src/chart.zoom.js

```diff
@@ -198,7 +198,13 @@ function doZoom(chartInstance, zoomX, zoomY, center, whichAxes) {
 		chartInstance.update(0);
 
 		if (typeof zoomOptions.onZoom === 'function') {
-			zoomOptions.onZoom();
+			zoomOptions.onZoom({
+				chartInstance: chartInstance,
```

Comment:
let's call this `chart` on the lefthand site. will match what we do in the main repo and is simpler

Reply:
same below for the pan callback